frontend/src/pages/modelServing/screens/global/InferenceServiceTableRow.tsx

```diff
@@ -22,6 +28,43 @@ const InferenceServiceTableRow: React.FC<InferenceServiceTableRowProps> = ({
   onEditInferenceService,
   isGlobal,
 }) => {
+  const renderAlert = () => {
+    switch (getInferenceServiceActiveModelState(inferenceService)) {
```

Comment:
I imagine we'd like to also get a message to go along with these -- but that may be a future UX thing to talk about. Error icons are weird without something to follow it -- and if we are doing it for errors, we should do it for everything I'd imagine.